lib/pinga-server/src/config.rs

```diff
@@ -184,7 +184,7 @@ fn default_concurrency_limit() -> usize {
 }
 
 fn default_layer_db_config() -> LayerDbConfig {
-    LayerDbConfig::default_for_service("pinga")
+    LayerDbConfig::default()
```

Comment:
What changed here? No service-specific defaults?

Reply:
Before we were feeding the service name to the disk cache path, but we were also generating a random path in that default like `default-cache-pinga-aj34s`, so passing the name was only useful if you needed to look at the specific disk cache of one of the services locally.